sdk/servicebus/azure-messaging-servicebus/migration-guide.md

```diff
@@ -334,9 +338,6 @@ Previously, you had the below options to receive messages from a session enabled
   an available set of sessions
 - Use the `ClientFactory.acceptMessageSessionAsync()` method to get an instance of the `IMessageSession` class that will be tied to a given sessionId or to the next available session if no sessionId is provided.
 
-While the first option is similar to what you would do in a non-session scenario, the second that allows you
-finer-grained control is very different from any other pattern used in the library.
```

Comment:
This made sense in .Net, but not for Java. We still are proposing an intermediate client, the only difference is that the final class is the same between sessions and non sessions.